sentry-raven/sentry-raven.gemspec

```diff
@@ -18,4 +18,11 @@ Gem::Specification.new do |gem|
   gem.executables = "raven"
 
   gem.add_dependency "faraday", ">= 1.0"
+
+  gem.post_install_message = <<~EOS
+    `sentry-raven` is deprecated! Please migrate to `sentry-ruby`
+
+    See. https://github.com/getsentry/sentry-ruby/tree/master/sentry-raven for details.
```

Comment:
can you change this into:

```
See https://docs.sentry.io/platforms/ruby/migration for the migration guide.
```

Reply:
@st0012 I fixed at d75d3c2 and edited description of PullRequest